recipes/AGDC/recipe.py

```diff
@@ -0,0 +1,41 @@
+"""Author: Norland Raphael Hagen - 09-16-2021
+Pangeo-Forge recipe for Australian Gridded Climate Data (AGCD) V2 precipitation, temperature and vapour pressure data"""  # noqa: E501
+
+from pangeo_forge_recipes.patterns import ConcatDim, FilePattern, MergeDim
+from pangeo_forge_recipes.recipes import XarrayZarrRecipe
+
+# Filename Pattern Inputs
+target_chunks = {"lat": 3451, "lon": 4426, "time": 20}
+years = list(range(1971, 2020))
+
+
+variables = ["precip", "tmax", "tmin", "vapourpres_h09", "vapourpres_h15"]
+
+
+def make_filename(variable, time):
+    if variable == "precip":
+        fpath = f"https://dapds00.nci.org.au/thredds/fileServer/zv2/agcd/v1/{variable}/total/r005/01day/agcd_v1_{variable}_total_r005_daily_{time}.nc"  # noqa: E501
+    else:
+        fpath = f"https://dapds00.nci.org.au/thredds/fileServer/zv2/agcd/v1/{variable}/mean/r005/01day/agcd_v1_{variable}_mean_r005_daily_{time}.nc"  # noqa: E501
+    return fpath
+
+
+# Preprocessing
+def preproc(ds):
+    ds = ds.drop_vars(["crs", "lat_bnds", "lon_bnds", "time_bnds"])
+    return ds
+
+
+pattern = FilePattern(
+    make_filename, ConcatDim(name="time", keys=years), MergeDim(name="variable", keys=variables)
+)
+
+
+# Recipe Inputs
+recipe = XarrayZarrRecipe(
+    file_pattern=pattern,
+    subset_inputs={"time": 2},
```

Comment:
So we are breaking each year into 2 pieces (approx 182 items in each) and then storing this to a target with chunk size 20. Overlaps between chunks on the year boundaries presumably mean that locking is required.

Reply:
```suggestion
```

Thanks for point this out, Ryan. Based on the size of these inputs, reported in https://github.com/pangeo-forge/staged-recipes/pull/87#issuecomment-1190752452, it looks like `subset_inputs` may not be necessary here. So I'm going to try running this without it.